src/components/options/CustomMacro.svelte

```diff
@@ -29,4 +34,11 @@
     <TextField dense bind:value={expansion}>Expansion</TextField>
   </Col>
   <Checkbox bind:checked={enabled} />
+  <!--
+    @Kento, @ChrRubin, can you see whether width should also be set to 3.5em?
+    Idk which looks better
+  -->
+  <Button icon class="red-text" style="height: 3.5em;" on:click={deleteMacro}>
+    <Icon path={mdiDelete} />
+  </Button>
```

Comment:
Its probably better to use a flex div to center it vertically than hardcoding a height. Right now it looks just slightly off center and the highlight shadow gets stretched.

Reply:
@KentoNishi refactored it to use a flex div